vita3k/config/include/config/config.h

```diff
@@ -50,7 +50,33 @@
     code(bool, "use-ubo", false, use_ubo)                                                               \
     code(bool, "performance-overlay", false, performance_overlay)                                       \
     code(std::string, "backend-renderer", "OpenGL", backend_renderer)                                   \
-    code(int, "user-id", 0, user_id)
+	code(int, "keyboard-button-select", 229, keyboard_button_select)									\
```

Comment:
Make sure to configure your IDE to use 4 spaces instead of tabs, it makes it looks weird and inconsistent in Github.
<img width="105" alt="Screen Shot 2020-03-29 at 1 14 53 PM" src="https://user-images.githubusercontent.com/32211852/77855604-6e36d180-71bf-11ea-8607-d156c0163600.png">

The format.sh script should've handled this... I wonder why it didn't pick it up...